src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ApplicationDbContext.cs

```diff
@@ -69,7 +69,7 @@ private enum IndicatorName
                     12, new Dictionary<FilterItemName, int>
                     {
                         {
-                            FilterItemName.School_Type__Total, 423
+                            FilterItemName.School_Type__Total, 727
```

Comment:
Double check the data because 727 is an ethnic filter for me in my Local and Development environments. 423 is the School Type Total for Subject 12.